__tests__/app.test.js

```diff
@@ -137,3 +139,68 @@ describe("5. GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("6. GET /api/articles/:article_id/comments", () => {
+  it("status:200, should responds with an array of comment objects", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSorted("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  it("status:404, should responds with error message when the path is not found", () => {
+    return request(app)
+      .get("/api/articles/1/commentsssssss")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Route not found");
+      });
+  });
+
```

Comment:
Please write another test of status 200 where the article ID exists but there are no comments, so need to expect an empty array.

Reply:
added underneath the first status 200 response